### DISCUSSION TITLE
Portuguese (Brazil) translation review

### DISCUSSION THREAD

**Discussion by allanmarum** [saleor/saleor#6295] [General]
We got 100% translated in storefront and core on Portuguese translation. 
https://www.transifex.com/mirumee/saleor-1/language/pt_BR/

But none of them were reviewed. Is it possible to setup a reviewer?

**Comment by karolkielecki** [comment]
@allanmarum Hi there! The link is broken... How does it work? We trust our community, so if something has been translated, we usually update the translations before the next releases.
